docs/02-install.md

```diff
@@ -1,9 +1,14 @@
-# Install
+# Basic Installations
 
-The installer provides a guided experience for provisioning the cluster on a
-particular platform. As of this writing, only AWS is a supported target.
+The scope of the new OpenShift 4 installer is purposefully narrow. It is
+designed for simplicity and ensured success. Many of the items and
+configurations that were previously handled by the installer are now expected
+to be "Day 1" operations, performed just after the installation of the
+control plane and basic workers completes. The installer provides a guided
```

Comment:
> `The installer provides a guided
experience for provisioning the cluster on a particular platform.`

this line has become redundant?

Reply:
It's unclear from the displayed snippet what makes it redundant. I don't think the fact that the experience is _guided_ (as opposed to non-interactive) is redundant -- this is the first time we talk about the process for execution of the installer.